server/controller/rideRequest.js

```diff
@@ -5,11 +5,8 @@ class RideRequestController{
         client.query("SELECT * FROM riderequests WHERE  rideId =$1", [ req.params.rideId ], (err, data) => {
```

Comment:
The same feedback I gave for getARide is happening here too, kindly follow the feedback and fix this too.